src/layman/layer/filesystem/tasks.py

```diff
@@ -23,7 +23,9 @@
     bind=True,
     base=celery_app.AbortableTask
 )
-def refresh_input_chunk(self, workspace, layername, check_crs=True, overview_resampling='', enable_more_main_files=False, time_regex=None, name_input_file_by_layer=None):
+def refresh_input_chunk(self, workspace, layername, check_crs=True, overview_resampling='',
+                        enable_more_main_files=False, time_regex=None, slugified_time_regex=None,
+                        name_input_file_by_layer=None):
```

Comment:
I suggest to add `assert (time_regex is None) == (slugified_time_regex is None)`